.github/workflows/test.yml

```diff
@@ -148,7 +148,7 @@ jobs:
           os: ubuntu-20.04
           arch: x86_64
           action: test
-          flags: --config=clang-tsan
+          flags: --config=clang-tsan --test_timeout=600
```

Comment:
https://github.com/dio/proxy-wasm-cpp-host/actions/runs/3182494962/jobs/5188568256#step:9:82 runtime test requires 312.0s

Reply:
Could you instead set `timeout = "large"` in Bazl rules for those two tests?